cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -45,11 +49,41 @@ const Modules = () => {
         setLoadingModules(false);
       });
   }, []);
+  useEffect(() => {
+    setFilteredData(
+      filteredData.filter((module: any) =>
+        displaymodule
+          .map((status) => status.toLowerCase())
+          .includes(module.status.toLowerCase()),
+      ),
+    );
+  }, [displaymodule]);
 
   const handleClick = () => {
     history("/modules/new");
   };
-
+  const handleSelectItem = (key: any) => {
+    let x = displaymodule.indexOf(key);
+    if (x > -1) {
+      setDisplayModule(displaymodule.filter((item) => item !== key));
+    } else {
+      setDisplayModule([...displaymodule, key]);
+    }
+  };
+  const resourceFilterPopover = () => {
+    return resourceFilter.map((item, index) => (
+      <Checkbox.Group
+        key={index}
+        style={{ display: "block" }}
+        onChange={(checkedValues) => handleSelectItem(item)}
+        value={displaymodule}
+      >
+        <>
+          <Checkbox value={item}>{item}</Checkbox>
+        </>
+      </Checkbox.Group>
```

Comment:
Does this create a new checkbox group for each resourceFilter? If yes, we should have a single group with multiple checkboxes

Reply:
changed according to the requirements